apps/playground/app/test-data-list/page.tsx

```diff
@@ -0,0 +1,228 @@
+                One after another
+              </Heading>
+              <DataListRoot>
+                <DataListItem>
+                  <DataListLabel>Appearance</DataListLabel>
+                  <DataListData>System</DataListData>
+                </DataListItem>
+                <DataListItem>
+                  <DataListLabel>Radius</DataListLabel>
+                  <DataListData>Medium</DataListData>
+                </DataListItem>
+              </DataListRoot>
+              <DataListRoot mt="4">
+                <DataListItem>
+                  <DataListLabel>Page background</DataListLabel>
+                  <DataListData>White</DataListData>
+                </DataListItem>
+                <DataListItem>
+                  <DataListLabel>Link color</DataListLabel>
+                  <DataListData>Blue</DataListData>
+                </DataListItem>
+              </DataListRoot>
+            </Box>
+
+            <Box mb="6">
+              <Heading mb="5" size="3">
+                With long label
+              </Heading>
+              <DataListRoot direction={{ initial: 'column', sm: 'row' }}>
+                <DataListItem>
+                  <DataListLabel>Name</DataListLabel>
+                  <DataListData>Vlad Moroz</DataListData>
+                </DataListItem>
+                <DataListItem>
+                  <DataListLabel>Email</DataListLabel>
+                  <DataListData>vlad@workos.com</DataListData>
+                </DataListItem>
+                <DataListItem>
+                  <DataListLabel>
+                    Lorem ipsum dolor sit amet consectetur adipscing elit
+                  </DataListLabel>
+                  <DataListData>
+                    <Link href="https://workos.com">WorkOS</Link>
+                  </DataListData>
+                </DataListItem>
+              </DataListRoot>
+            </Box>
+
+            <Box mb="6">
+              <Heading mb="5" size="3">
+                With varied X & Y gaps
+              </Heading>
+              <DataListRoot direction={{ initial: 'column', sm: 'row' }} gapX="6" gapY="2">
+                <DataListItem>
+                  <DataListLabel>Name</DataListLabel>
+                  <DataListData>Vlad Moroz</DataListData>
+                </DataListItem>
+                <DataListItem>
+                  <DataListLabel>Email</DataListLabel>
+                  <DataListData>vlad@workos.com</DataListData>
+                </DataListItem>
+                <DataListItem>
+                  <DataListLabel>Lorem ipsum dolor</DataListLabel>
+                  <DataListData>
+                    <Link href="https://workos.com">WorkOS</Link>
+                  </DataListData>
+                </DataListItem>
+              </DataListRoot>
+            </Box>
+
+            <Box mb="6">
+              <Heading mb="5" size="3">
+                With custom `columns`
+              </Heading>
+              <DataListRoot columns="100px 1fr">
```

Comment:
Since we're using subgrid, really long labels naturally will grow the width of the label column. I think the use of `columns` will be an edge case since the grid / subgrid interaction feels really nice / responsive as a default.

Reply:
After playing with `grid/subgrid` I didn't feel that setting `width` on the label children made sense anymore and passing `columns` felt more appropriate. Of course this would only apply to the `row` direction, so we'd need to document that. Open to feedback on this!